src/graph_compute/phase_scheduling.h

```diff
@@ -117,7 +128,7 @@ static void createChunkData(vertex_t * const nodes, const vid_t cntNodes,
   cilk_for (vid_t i = 0; i < scheddata->cntChunks; ++i) {
     chunkdata_t * chunk = &scheddata->chunkdata[i];
     chunk->nextIndex = i << CHUNK_BITS;
-    chunk->phaseEndIndex[0] = std::min(chunk->nextIndex + (1 << (CHUNK_BITS - 1)),
+    chunk->phaseEndIndex[0] = std::min(chunk->nextIndex + ((1 << CHUNK_BITS) >> 1),
```

Comment:
True, though I don't think this is where the error should come from (i.e., some technical calculation deep in the initialization code).  Instead, we should probably have a preprocessor error at the top of phase_scheduler.h.